native/c/lexer.hpp

```diff
@@ -967,14 +965,13 @@ class Lexer
   static bool is_ident_start(char c)
   {
     unsigned char uc = static_cast<unsigned char>(c);
-    return (uc >= 'A' && uc <= 'Z') || (uc >= 'a' && uc <= 'z') ||
-           c == '$' || c == '_' || c == '/' || c == '*';
+    return uc != ' ' && iscntrl(uc) == 0;
   }
 
-  static bool is_ident_cont(char c)
+  static bool
+  is_ident_cont(char c)
```

Comment:
This looks weird to split across 2 lines, but I guess it's valid syntax 😋

Reply:
529ad74